src/VideoStreaming/VideoReceiver.cc

```diff
@@ -205,7 +205,7 @@ void VideoReceiver::start()
             }
             g_object_set(G_OBJECT(dataSource), "uri", qPrintable(_uri), "caps", caps, NULL);
         } else {
-            g_object_set(G_OBJECT(dataSource), "location", qPrintable(_uri), "latency", 0, "udp-reconnect", 1, "timeout", 5000000, NULL);
+            g_object_set(G_OBJECT(dataSource), "location", qPrintable(_uri), "latency", 17, "udp-reconnect", 1, "timeout", static_cast<guint64>(5000000), NULL);
```

Comment:
17ms helps, obviously a whole frame is going to be more accommodating of jittery networks. No, most people won't notice a single frame delay. Regardless on these sorts of links TCP-based streams are pretty much out the window and all we can feasibly solve here is packet jitter and reordering. 

I have no objections to increasing 33-35ms range, though we've apparently gotten away with no jitter buffer for a while which tells me either few people are using this feature or RTSP streaming setups are using pretty robust wireless links.


The gobject type issue _couldn't_ be caught by the compiler since it is a variadic function. I'm not aware of any MSVC probes or guards we could have turned on to catch this.